src/Components/Input.re

```diff
@@ -183,8 +181,9 @@ let%component make =
     let dimensions =
       Revery_Draw.Text.measure(
         ~smoothing=Revery.Font.Smoothing.default,
-        ~fontFamily=Styles.fontFamily,
-        ~fontSize=Styles.fontSize,
+        ~fontFamily=
+          Revery.Font.Family.toPath(fontFamily, Normal, false, false),
```

Comment:
Unrelated to this but I should change this function to take the `FontFamily.t` with the weight, italicized, and monospaced as separate arguments.

Reply:
Yeah, that's almost the only complaint I have with the API. That and the unlabelled boolean arguments, which isn't very intuitive.